components/match2/wikis/counterstrike/match_group_input_custom.lua

```diff
@@ -390,11 +391,14 @@ function matchFunctions.getEarnings(name, year)
 		return 0
 	end
 
-	return tonumber(EarningsOf._team(name, {sdate = year .. '-01-01', edate = year .. '-12-31'}))
+	return tonumber(EarningsOf._team(name, {sdate = (year-1) .. '-01-01', edate = year .. '-12-31'}))
 end
 
 function matchFunctions.isFeatured(match)
-	if Table.includes(FEATURED_TIERS, match.liquipediatier) then
+	if
+		tonumber(match.liquipediatier or '') == S_TIER
+		or tonumber(match.liquipediatier or '') == A_TIER
+	then
```

Comment:
```suggestion
	if Table.includes(FEATURED_TIERS, tonumber(match.liquipediatier)) then
```

plus in line 30:
`local FEATURED_TIERS = {'1', '2'}`

Reply:
I created the to globals to add a meaning to the `1` and `2`

Reply:
1 and 2 mean tier 1/tier 2 so  imo reasonable to use them
also S-TIer / A-Tier are only the display values

if wanted you could do something like this in line 30:
```lua
local S_TIER = 1
local A_TIER = 2
local FEATURED_TIERS = {S_TIER, A_TIER}
```

Reply:
Okay I will just use `FEATURED_TIERS = {1, 2}`